root/static/scripts/common/constants/locales.json

```diff
@@ -720,8 +761,6 @@
   "vai_Vaii_LR":"Vai Liberia Vai",
   "vi":"Vietnamese",
   "vi_VN":"Vietnamese Vietnam",
-  "vo":"Volapük",
-  "vo_001":"Volapük World",
```

Comment:
These seem unused, although it's a bit strange that they'd remove them while leaving other made up languages in :man_shrugging: .

Reply:
It has been removed from “common” locales in [CLDR 38](https://cldr.unicode.org/index/downloads/cldr-38) because the coverage level was not enough, but it is still listed in “seed” locales. I reported the issue to `DateTime::Locale`.